transport/session/src/lib.rs

```diff
@@ -14,7 +14,7 @@ pub use manager::{DispatchResult, SessionManager, SessionManagerConfig};
 use crate::types::SessionTarget;
 use hopr_network_types::prelude::state::SessionFeature;
 pub use hopr_network_types::types::*;
-use libp2p_identity::PeerId;
+use hopr_primitive_types::prelude::Address;
```

Comment:
Maybe the transport package itself should use `PeerId`, since that is it's native type.

Reply:
We can unify this later.